kubernetes-1.25.yaml

```diff
@@ -182,17 +182,42 @@ subpackages:
           CFLAGS="$CFLAGS -static -DVERSION=${{vars.pause-version}}-${{package.version}}"
           gcc -o "${{targets.subpkgdir}}"/usr/bin/pause-3.8 linux/pause.c
 
-  - name: kubernetes-1.25-default
-    description: "Compatibility package to set 1.25 as the default kubernetes, and add packages to their shortened path"
+  - range: components
+    name: "default-${{range.key}}-1.25"
+    description: "Makes this version of ${{range.key}} the default."
+    dependencies:
+      runtime:
+        - ${{range.key}}-1.25
+      provides:
+        - default-${{range.key}}=1.25
     pipeline:
       - runs: |
           mkdir -p ${{targets.subpkgdir}}/usr/bin
-          ln -s kubectl-1.25 ${{targets.subpkgdir}}/usr/bin/kubectl
-          ln -s kubelet-1.25 ${{targets.subpkgdir}}/usr/bin/kubelet
-          ln -s kube-scheduler-1.25 ${{targets.subpkgdir}}/usr/bin/kube-scheduler
-          ln -s kube-proxy-1.25 ${{targets.subpkgdir}}/usr/bin/kube-proxy
-          ln -s kube-controller-manager-1.25 ${{targets.subpkgdir}}/usr/bin/kube-controller-manager
-          ln -s kube-apiserver-1.25 ${{targets.subpkgdir}}/usr/bin/kube-apiserver
+          ln -s ${{range.key}}-1.25 ${{targets.subpkgdir}}/usr/bin/${{range.key}}
+
+  - name: kubernetes-1.25-default
```

Comment:
this feels a little nitty, but do we want `default-kubernetes-1.25` instead to match the component defaults? or vice versa for some sort of standardization?

Reply:
If anything I'd flip the others around because this package isn't new (see L185 in the baseline).

I biased this way because we also have `default-jvm` and friends in JRE land, but it's probably better to be consistent with things more locally in this case.

I wish we had better guidance on naming packages because I feel like we have loads of little inconsistencies like this. 😞